src/client/Microsoft.Identity.Client/SSHCertificates/SSHExtenions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.TelemetryCore.Internal.Events;
+
+namespace Microsoft.Identity.Client.SSHCertificates
+{
+    /// <summary>
+    /// Extensions that add support for SSH certificates
+    /// </summary>
+    public static class SSHExtenions
+    {
+        /// <summary>
+        /// Instructs AAD to return an SSH certificate instead of a Bearer token. The SSH certificate 
+        /// (not the same as public / private key pair used by SSH), can be used to securely deploy 
+        /// a public SSH key to a machine. See https://aka.ms/msal-net-shh for details.
+        /// </summary>
+        /// <param name="builder">Interactive authentication builder</param>
+        /// <param name="publicKeyJwk">The public SSH key in JWK format (https://tools.ietf.org/html/rfc7517). 
+        /// Currently only RSA is supported, and the JWK should contain only the RSA modulus and exponent</param>
+        /// <param name="keyId">A key identifier, it can be in any format. Used to distinguish between 
+        /// different keys when fetching an SSH certificate from the token cache.</param>
+        public static AcquireTokenInteractiveParameterBuilder WithSSHCertificateAuthenticationScheme(
```

Comment:
Is it expected to only do it for Interfactive? (and not `AbstractPublicClientAcquireTokenParameterBuilder<T>`)? is it a limitation of EVO? for instance would that work with Device Code Flow or IWA ? (I would think so)

Do we want to add an ak.ms link for a new wiki page for SSHCert?

Reply:
Yes and Yes :) It is expected to only work for Interactive and Silent, i.e. authorization and refresh_token. Will add a link

Reply:
Oh I thought this would be a feature only used by AzureCLI and PowerShell, so we don't necessarily need to make it more discoverable by providing wiki and aka.ms links. (Otherwise app devs would notice it and ask "what is this for", and then our response would be "[this is not the droid you are looking for](https://www.youtube.com/watch?v=GO_xfR64qSk)".)